content/en/llm_observability/terms/_index.md

```diff
@@ -148,11 +148,89 @@ You can instrument your LLM application to trace and group together all workflow
+
+| Evaluation Stage | Evaluation Method | Evaluation Definition | 
+|---|---|---|
+| Evaluated on Output | Evaluated using LLM | Failure To Answer flags whether each prompt-response pair demonstrates that the LLM application has provided a relevant and satisfactory answer to the user's question.  |
+
+#### Language Mismatch
+
+This check identifies instances where the LLM generates responses in a different language or dialect than the one used by the user, which can lead to confusion or miscommunication. This check ensures that the LLM's responses are clear, relevant, and appropriate for the user's linguistic preferences and needs.
+
+{{< img src="llm_observability/evaluations/language_mismatch.png" alt="A Language Mismatch evaluation detected by an open source model in LLM Observability" style="width:100%;" >}}
+
+| Evaluation Stage | Evaluation Method | Evaluation Definition | 
+|---|---|---|
+| Evaluated on Input and Output | Evaluated using Open Source Model | Language Mismatch flags whether each prompt-response pair demonstrates that the LLM application answered the user's question in the same language that the user used.  |
+
+#### Sentiment
+
+This check helps understand the overall mood of the conversation, gauge user satisfaction, identify sentiment trends, and interpret emotional responses. This check accurately classifies the sentiment of the text, providing insights to improve user experiences and tailor responses to better meet user needs.
+
+{{< img src="llm_observability/evaluations/sentiment.png" alt="A Sentiment evaluation detected by an LLM in LLM Observability" style="width:100%;" >}}
+
+| Evaluation Stage | Evaluation Method | Evaluation Definition | 
+|---|---|---|
+| Evaluated on Input and Output | Evaluated using LLM | Sentiment flags the emotional tone or attitude expressed in the text, categorizing it as positive, negative, or neutral.   |
+
+### Security and Safety evaluations
+
+#### Toxicity
+
+This check evaluates each input prompt from the user and the response from the LLM application for toxic content. This check identifies and flags toxic content to ensure that interactions remain respectful and safe.
+
+{{< img src="llm_observability/evaluations/toxicity.png" alt="A Toxicity evaluation detected by an LLM in LLM Observability" style="width:100%;" >}}
+  
+| Evaluation Stage | Evaluation Method | Evaluation Definition | 
+|---|---|---|
+| Evaluated on Input and Output | Evaluated using LLM | Toxicity flags any language or behavior that is harmful, offensive, or inappropriate, including but not limited to hate speech, harassment, threats, and other forms of harmful communication. |
+
+#### Prompt Injection
+
+This check identifies attempts to manipulate the LLM's responses or redirect the conversation in ways not intended by the original users. This check maintains the integrity and authenticity of interactions between users and the LLM.
```

Comment:
Is this about a third party or is this about the original intent by the designer? I'm a bit confused.